content/010_introduction/16_workshop_prerequisites.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Workshop Prerequisites"
+chapter: true
+weight: 16
+---
+
+## Workshop Prerequisites
+
+There are a few prerequisite tasks you must perform before getting started on this workshop. You will integrate various tools and services into a CI/CD pipeline which will require you to create or register new accounts with the following services:
+
+1. Create a [GitHub account][1]
+1. Create a [Docker Hub account][2]
+1. Create a [Synk account][3]
+1. Create a [CircleCI account][4]
+1. Create a [Terraform Cloud account][5]
+1. Create an [AWS account][6]
+<!-- 1. Create a Cloud9 IDE Workspace -->
```

Comment:
@eugenemu I commented out the Cloud9 stuff here because I figured that you will cover this in the AWS setup section.

Reply:
Yeah no worries, I'll merge this and when I make my updates, will remove it